tests/test_iib_ops.py

```diff
@@ -338,6 +377,53 @@ def test_remove_operators_cli(
     ]
 
 
+def test_remove_operators_cli_skip_pulp(
```

Comment:
Would not be good idea to use the code above (`test_remove_operators_cli`) and just use `@pytest.mark.parametrize` to set the `--skip-pulp` and expected result? It looks like the core of both tests is the same.